### DISCUSSION TITLE
Trans-effect caculation

### DISCUSSION THREAD

**Discussion by nhu-github** [Katsevich-Lab/sceptre#131] [Q&A] [answered]
If a long list of genes, say 20k genes, that I would like to run trans effect caculation. So there is going to be 20k*20k pairs to investigate. And this is super computational exprensive, maybe 80days. Is there a way to reduce the time comsuption? I am interested in if any folks met this situation before and came up with any situations. Thanks!!!

**Comment by ekatsevi** [comment, marked as answer]
If you have access to high-performance computing resources, I recommend you try our Nextflow pipeline, which is documented in [Part II of our e-book](https://timothy-barry.github.io/sceptre-book/at-scale.html). This should greatly accelerate your analysis.
